packages/react/src/components/TimePicker/next/TimePicker-test.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { default as TimePicker } from './TimePicker';
+
+import { render, screen, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+describe('TimePicker', () => {
+  describe('input', () => {
+    it('renders as expected', () => {
+      render(<TimePicker id="time-picker" />);
+      expect(screen.getByRole('textbox')).toBeInTheDocument();
+    });
+
+    it('passes classNames as expected', () => {
+      render(<TimePicker id="time-picker" className="🚀"></TimePicker>);
+      expect(screen.getByRole('textbox')).toHaveClass('🚀');
+    });
+
+    it('should set type as expected', () => {
+      render(<TimePicker id="time-picker"></TimePicker>);
+      expect(screen.getByRole('textbox')).toHaveAttribute('type', 'text');
+    });
+
+    it('should set value as expected', () => {
+      render(<TimePicker id="time-picker" value="🐶"></TimePicker>);
+      expect(screen.getByRole('textbox')).toHaveAttribute('value', '🐶');
+    });
+
+    it('should set disabled as expected', () => {
+      const onClick = jest.fn();
+      render(
+        <TimePicker id="time-picker" onClick={onClick} disabled></TimePicker>
+      );
+      fireEvent.click(screen.getByRole('textbox'));
+      expect(onClick).not.toHaveBeenCalled();
+    });
+
+    it('should set placeholder as expected', () => {
+      render(<TimePicker id="time-picker" placeholder="🧸"></TimePicker>);
+      expect(screen.getByPlaceholderText('🧸')).toBeInTheDocument();
+    });
+  });
+  describe('label', () => {
```

Comment:
Style nit for spacing

```suggestion

  describe('label', () => {
```